samples/crypto/aes_ctr/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _crypto_aes_ctr:
+
+Crypto: AES CTR
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The AES sample shows how to perform AES encryption and decryption operations using the CTR mode without padding using an 128 bit AES key.
+
+It does the following things:
```

Comment:
I feel this sentence is redundant.

```suggestion
First, the sample encrypts a sample plaintext:
```
and then provide the steps in lines 18-22 as an ordered list. Remove text in line 17.